constructor/osx/prepare_installation.sh

```diff
@@ -31,8 +34,7 @@ touch "$PREFIX/conda-meta/history"
 # Extract the conda packages but avoiding the overwriting of the
 # custom metadata we have already put in place
 notify "Preparing packages..."
-"$CONDA_EXEC" constructor --prefix "$PREFIX" --extract-conda-pkgs
-if (( $? )); then
+if ! "$CONDA_EXEC" constructor --prefix "$PREFIX" --extract-conda-pkgs; then
```

Comment:
Does this need grouping in the command or is this an idiomatic shell expression? Sorry about my ignorance, just trying to understand better :D

Reply:
see https://www.shellcheck.net/wiki/SC2181